app/controllers/bulk_tagging/collections_controller.rb

```diff
@@ -0,0 +1,15 @@
+module BulkTagging
```

Comment:
Would it be clearer for now to merge this and `UpdateTagsController`? Feels like there would be lots of small controllers if we specify one per feature (topics/browse/search etc).